Source/Android/jni/MainAndroid.cpp

```diff
@@ -107,6 +107,10 @@ void Host_UpdateMainFrame()
 
 void Host_RequestRenderWindowSize(int width, int height)
 {
+  // Update touch pointer
+  JNIEnv* env;
+  IDCache::GetJavaVM()->AttachCurrentThread(&env, nullptr);
```

Comment:
Does this need a corresponding DetachCurrentThread()?

Reply:
Fixed